beacon-chain/state/state-native/setters_misc.go

```diff
@@ -41,6 +41,11 @@ const (
 	indicesLimit = 8000
 )
 
+// SetId set the identifier of the beacon state.
+func (b *BeaconState) SetId(id uint64) {
+	b.id = id
+}
```

Comment:
Do we need to add lock here?

Reply:
This is only useful for tests, so I don't think so. I will think of a way to not need this setter at all.